docs/using-injected-bindings/using-injected-bindings.md

```diff
@@ -0,0 +1,165 @@
+field in the binding Secret resource.  Normally `type` field should be good
+enough to look up the bindings.  The provider field could be used where there
+are different providers for the same Provisioned Service type.  For example, if
+the type is `mysql`, the provider value could be `mariadb`, `oracle`, `bitnami`,
+`aws-rds`, etc.  When the application is reading the binding values, if
+necessary, the application could consider `type` and `provider` as a composite
+key to avoid ambiguity.  It will be helpful if an application needs to choose a
+particular provider based on the deployment environment.  In the deployment
+environment (`stage`, `prod`, etc.), at any given time, you need to ensure only
+one bindings projection exits for a given `type` or `type` & `provider` --
+unless your application needs to connect to all the services.
+
+## Using Binding injected as files
+
+For determining the folder where bindings are injected, you can check the 
+`.spec.mountPath` value from the `ServiceBinding` resource which specifies the destination used for the injection.
+
+Alternatively, you can also use `SERVICE_BINDING_ROOT` environment variable in the application resource.  
+If both are set then the
+`SERVICE_BINDING_ROOT` environment variable takes the higher precedence.
+
+The following table summarizes how the final bind path is computed:
+
+| .spec.mountPath | SERVICE_BINDING_ROOT  | Final Bind Path                      |
+| --------------- | --------------------- | ------------------------------------ |
+| nil             | non-existent          | /bindings/ServiceBinding_Name        |
+| nil             | /some/path/root       | /some/path/root/ServiceBinding_Name  |
+| /home/foo       | non-existent          | /home/foo                            |
+| /home/foo       | /some/path/root       | /some/path/root/ServiceBinding_Name  |
+
+You can use the built-in language feature of your programming language of choice
+to read environment variables.
+
+For accessing bindings within `SERVICE_BINDING_ROOT`, there are language or
+framework specific programs:
+
+1. Python: https://github.com/baijum/pyservicebinding
+2. Go: https://github.com/baijum/servicebinding
+3. Java/Spring: https://github.com/spring-cloud/spring-cloud-bindings
+4. Quarkus: https://quarkus.io/guides/deploying-to-kubernetes#service-binding
+5. JS/Node: https://github.com/nodeshift/kube-service-bindings
+
+Note: All these libraries expect `SERVICE_BINDING_ROOT` to be set.
+
+Here is example Python client usage:
+
+```
+from pyservicebinding import binding
+try:
+    sb = binding.ServiceBinding()
+except binding.ServiceBindingRootMissingError as msg:
+    # log the error message and retry/exit
+    print("SERVICE_BINDING_ROOT env var not set")
+
+sb = binding.ServiceBinding()
+
+bindings_list = sb.bindings("postgresql")
+```
+
+In the above example, the `bindings_list` contains bindings for `postgresql`
```

Comment:
- above ---> previous
- `bindings_list`---> `bindings_list` parameter
- bindings ----> the binding metadata
- `postgresql`---> the `postgresql`